gen3/bin/healthcheck.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Finds any unhealthy pods and nodes
+#
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+help() {
+  gen3 help healthcheck
+}
+
+gen3_healthcheck() {
+  local HEALTH_SEND_SLACK=false
+  while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+      '--slack')
+	HEALTH_SEND_SLACK=true
+	shift
+	;;
+      *)
+	echo "Unrecognized flag"
+	help
+	exit 1
+	;;
+    esac
+  done
+
+  # refer to k8s api docs for pod status info
+  # https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#podstatus-v1-core
+  local allPods=$(g3kubectl get pods -o json | \
```

Comment:
Pass `--all-namespaces` (I think that's it), so we get everything on the cluster (jupyterhub namespaces, kube-system, etc)